app/src/main/res/values/strings.xml

```diff
@@ -236,13 +236,12 @@
   <string name="upload_image_too_dark">This picture is too dark, are you sure you want to upload it? Wikimedia Commons is only for pictures with encyclopedic value.</string>
   <string name="upload_image_blurry">This picture is blurry, are you sure you want to upload it? Wikimedia Commons is only for pictures with encyclopedic value.</string>
 
-  <string name="upload_image_problem_duplicate">This file already exists on Commons. Are you sure you want to proceed?</string>
-  <string name="upload_image_problem_dark">This picture is too dark, are you sure you want to upload it? Wikimedia Commons is only for pictures with encyclopedic value.</string>
-  <string name="upload_image_problem_blurry">This picture is blurry, are you sure you want to upload it? Wikimedia Commons is only for pictures with encyclopedic value.</string>
-  <string name="upload_image_problem_dark_duplicate">This picture is too dark and already exists on Commons, are you sure you want to upload it? Please only upload pictures with encyclopedic value.</string>
-  <string name="upload_image_problem_blurry_duplicate">This picture is blurry and already exists on Commons, are you sure you want to upload it? Please only upload pictures with encyclopedic value.</string>
-  <string name="upload_image_problem_dark_blurry">This picture is too dark and is blurry, are you sure you want to upload it? Please only upload pictures with encyclopedic value.</string>
-  <string name="upload_image_problem_dark_blurry_duplicate">This picture is too dark, is blurry and already exists on Commons, are you sure you want to upload it? Please only upload pictures with encyclopedic value.</string>
+  <string name="upload_problem_exist">This image might not be proper to be uploaded:</string>
+  <string name="upload_problem_image_dark">\n - Image is too dark.</string>
+  <string name="upload_problem_image_blurry">\n - Image is blurry.</string>
+  <string name="upload_problem_image_duplicate">\n - Image is already on Commons.</string>
+  <string name="upload_problem_different_geolocation">\n - This picture is taken at another location than you attempt to upload to.</string>
+  <string name="upload_problem_do_you_continue">\n Do you still want to upload this picture?</string>
```

Comment:
Remove whitespace before "Do", otherwise it shows up in the dialog. I also suggest an additional newline before this sentence.